src/components/include/DefaultImportModalDialog.jsx

```diff
@@ -0,0 +1,323 @@
+import nativeFormats from "@exabyte-io/made.js/lib/parsers/native_formats";
```

Comment:
we already have TypeScript in this project, preferable to write new components using .ts/.tsx extensions.

Reply:
Where exactly is TS in Materials Designer? I don't think we are using it here yet